evm/src/libraries/EndpointHelpers.sol

```diff
@@ -35,3 +35,13 @@ function isFork(uint256 evmChainId) view returns (bool) {
 function min(uint256 a, uint256 b) pure returns (uint256) {
     return a < b ? a : b;
 }
+
+// @dev Count the number of set bits in a uint64
+function countSetBits(uint64 x) pure returns (uint8 count) {
+    while (x != 0) {
+        x &= x - 1;
+        count++;
+    }
+
+    return count;
+}
```

Comment:
can we move this to a separate utils library or rename EndpointHelpers -> NTTUtils.sol

Reply:
I've opened https://github.com/wormhole-foundation/example-native-token-transfers/issues/183 to do this, distracts from the goal of this PR imo